templates/desktop/snappy.html

```diff
@@ -1,183 +1,185 @@
-{% extends "cloud/base_cloud.html" %}
+{% extends "cloud/base_cloud-v1.html" %}
 
 {% block title %}Snaps on Ubuntu | Desktop{% endblock %}
 
 {% block meta_description %}Introducing snaps on Ubuntu, the new way to install, update and find apps.{% endblock meta_description %}
 
 {% block second_level_nav_items %}
-<div class="strip-inner-wrapper">
-  {% include "templates/_nav_breadcrumb.html" with section_title="Desktop" page_title="Snaps on Ubuntu" %}
-</div>
+  {% include "templates/_nav_breadcrumb-v1.html" with section_title="Desktop" page_title="Snaps on Ubuntu" %}
 {% endblock second_level_nav_items %}
 
 {% block content %}
-<section class="row row-hero row-grey">
-  <div class="strip-inner-wrapper equal-height">
-    <div class="six-col equal-height__item">
+<div class="p-strip--grey is-deep is-bordered">
+  <div class="row">
+    <div class="col-7">
       <h1>A &lsquo;snap&rsquo; is a universal Linux package</h1>
       <p>Snaps work on any distribution or device. Snaps are faster to install, easier to create, safer to run, and they update automatically and transactionally so your app is always fresh and never broken.</p>
 
       <p>The public collection of snaps includes the latest and best apps from GitHub and beyond, so you have the whole world of Linux apps at your fingertips.</p>
-      <p><a class="button--primary" href="http://snapcraft.io/docs/core"><span class="external">Get started with snaps</span></a></p>
+      <p><a class="p-button--brand p-link--external" href="http://snapcraft.io/docs/core"><span class="external">Get started with snaps</span></a></p>
```

Comment:
this should be

```html
<p><a class="p-button--brand" href="http://snapcraft.io/docs/core"><span class="p-link--external">Get started with snaps</span></a></p>
```
![image](https://cloud.githubusercontent.com/assets/441217/26699168/aadd52c6-4710-11e7-8259-ceece74a8841.png)